src/components/include/utils/logger2.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <log4cxx/logger.h>
+#include <string>
+#include "utils/ilogger.h"
+
+namespace logger {
+class Log4CXXLogger;
+}
+
+typedef logger::Log4CXXLogger ExternalLogger;
+
+#define LOCATTION_INFO LOG4CXX_LOCATION
```

Comment:
Are you using SDL location info or Log4cXX location info?

Reply:
removed